modules/nf-core/upd/main.nf

```diff
@@ -0,0 +1,36 @@
+
+process UPD {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "bioconda::upd=0.1.1"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/upd:0.1.1--pyhdfd78af_0':
+        'biocontainers/upd:0.1.1--pyhdfd78af_0' }"
+
+    input:
+        tuple val(meta), path(vcf)
+
+    output:
+        tuple val(meta), path("*.bed"), emit: bed
+        path "versions.yml"           , emit: versions
+
+    when:
+        task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    """
+    upd \\
+        --vcf $vcf \\
+        $args \\
+        --af-tag AF \\
+        regions --out ${prefix}.bed
```

Comment:
The --af-tag AF should be in tests/modules/nf-core/upd/nextflow.config otherwise won't work when the input comes from vep, for example. By having it in the nextflow.config you can modify it so that it can run with different types of vcfs

Reply:
this has been fixed